addons/point_of_sale/tests/test_pos_invoice_consolidation.py

```diff
@@ -0,0 +1,113 @@
+from odoo.tests.common import TransactionCase
+from odoo import Command
+
+
+class TestPosInvoiceConsolidation(TransactionCase):
```

Comment:
You should always inherit common classes like `TestPointOfSaleCommon` so we not repeat code, Also check another common classes for sale